input/pagecontent/ch-mhd-1.md

```diff
@@ -22,15 +22,15 @@ The Update Document Metadata [CH-MHD-1] transaction is used to update document m
 
 #### Update Document Metadata Request Message
 
-The Update Document Metadata Request Message provides the ability to submit updated attributes for a DocumentReference resource in a community.
+The Update Document Metadata Request Message provides the ability to submit updated attributes for DocumentReference resources.
 
 ##### Trigger Events
 
 The Update Document Metadata Request Message is triggered when a Document Source needs to transmit updated DocumentReference metadata.
 
 ##### Message Semantics
 
-A Document Source initiates a FHIR request using Update as defined at http://hl7.org/fhir/http.html#update on a DocumentReference Resource.
+A Document Source initiates a FHIR request using Update as defined at http://hl7.org/fhir/http.html#update on DocumentReference Resources.
```

Comment:
I like the wording of ITI-130:

A Document Source initiates a FHIR request using Update as defined at http://hl7.org/fhir/http.html#update on DocumentReference Resources, with a standalone HTTP request or a [transaction](https://hl7.org/fhir/R4/http.html#transaction).